specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+        "bandwidthMHz",
+        "direction",
+        "polarization"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink).",
+          "enum": [
+            "uplink",
+            "downlink"
+          ],
+          "x-ms-enum": {
+            "name": "direction",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "authorizations": {
+          "type": "array",
+          "description": "Authorized Ground Stations",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AuthorizedGroundstation"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "List of authorized spacecraft links per ground station and the expiration date of the authorization."
+    },
+    "Spacecraft": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SpacecraftsProperties",
+          "description": "Spacecraft Properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SpacecraftsProperties"
+            }
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Customer creates a spacecraft resource to schedule a contact."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "x-ms-azure-resource": true,
```

Comment:
I am puzzled about this, I don't think we are supposed to mark ResourceReference with x-ms-azure-resource as true. Please everyone let me know if I am misinformed.

Reply:
Can you try removing this?

Reply:
done

Reply:
The docs could be clearer, but I think x-ms-azure-resource is mainly for types that are supposed to represent the resource payload for TrackedResource/ProxyResource, and so it suitable for deriving from some common 'resource' base type in SDKs.

Reply:
@allenjzhang @raych1 Do you know if I understand it correctly?